cypress/e2e/awx/resources/workflowJobTemplates.cy.ts

```diff
@@ -0,0 +1,129 @@
+import { randomString } from '../../../../framework/utils/random-string';
+import { Inventory } from '../../../../frontend/awx/interfaces/Inventory';
+import { Label } from '../../../../frontend/awx/interfaces/Label';
+import { Organization } from '../../../../frontend/awx/interfaces/Organization';
+
+describe('Workflow Job templates form', () => {
+  let organization: Organization;
+
+  let inventory: Inventory;
+
+  // let githubTokenCrendential: Credential;
+  let label: Label;
+
+  before(() => {
+    cy.awxLogin();
+
+    cy.createAwxOrganization().then((o) => {
+      organization = o;
+      cy.createAwxInventory({ organization: organization.id }).then((i) => {
+        inventory = i;
+      });
+      cy.createAwxLabel({ organization: organization.id }).then((l) => {
+        label = l;
+      });
+    });
+  });
+
+  after(() => {
+    cy.deleteAwxOrganization(organization);
+  });
+
+  it.skip('Should create job template with all fields except for prompt on launch values', () => {
+    const jtName = 'E2E ' + randomString(4);
+
+    cy.navigateTo(/^Templates$/);
+    cy.clickButton(/^Create template$/);
+    cy.clickLink(/^Create workflow job template$/);
+
+    // Name
+    cy.typeInputByLabel(/^Name$/, jtName);
+
+    // Description
+    cy.typeInputByLabel(/^Description$/, 'this is a description');
+
+    cy.selectDropdownOptionByLabel(/^Labels$/, label.name.toString(), true);
+
+    // Job tags
+    cy.contains('.pf-c-form__label-text', /^Job tags$/)
```

Comment:
There are cy custom commands for handling this.
using a bunch of cy command like this is going to make upgrading to PF5 hard.